extensions/wikia/AdEngine/js/provider/gpt/googleSlots.js

```diff
@@ -0,0 +1,25 @@
+define('ext.wikia.adEngine.provider.gpt.googleSlots', function() {
```

Comment:
Sounds like it'd be too much unnecessary noise for now. If we find it useful in future, we'll be able to add it - it's not a big thing, right? :)